config/environments/production.rb

```diff
@@ -93,7 +88,7 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
-  config.i18n.fallbacks = [I18n.default_locale]
+  config.i18n.fallbacks = true
```

Comment:
The `i18n` gem has/had a post install message saying the `true` was deprecated here, and that an array of locale should be used instead. I think we need to keep the deleted version of this line.